app/src/main/java/skills/future/planer/db/task/TaskDataRepository.java

```diff
@@ -44,7 +44,6 @@
         AppDatabase db = AppDatabase.getInstance(application);
         taskDataDao = db.taskDataTabDao();
         listLiveData = taskDataDao.getTaskData();
-        new SelectAsyncTask(taskDataDao, this).execute();
```

Comment:
Mieliśmy dodawać a nie usuwać wątki XD

Reply:
XDD

Reply:
żeby zrobić to na wątkach muszę przerobić bazę danych